src/main/java/com/sourcery/gymapp/backend/userProfile/consumer/UserProfileKafkaConsumer.java

```diff
@@ -33,4 +36,19 @@ public void onMessage(ConsumerRecord<UUID, String> record) {
             KafkaProcessingContext.disableKafkaProcessing();
         }
     }
+
+    @KafkaListener(topics = "${spring.kafka.topics.likes-notifications}", groupId = "user-profile-listener-group")
+    public void onLikesNotification(ConsumerRecord<UUID, String> record) {
+        try {
+            KafkaProcessingContext.enableKafkaProcessing();
+
+            var data = objectMapper.readValue(record.value(), LikeNotificationEvent.class);
+            likeNotificationService.uploadLikeNotifications(data);
```

Comment:
we might want to check that the amount of likes is not negative (could lead to weird UI behaviour?)

Reply:
It's intended behaviour, on th UI side we should if something lost or got more/less likes:
![image](https://github.com/user-attachments/assets/798da986-ceba-4b2a-b0be-f03660c955bc)